config/vufind/KohaRest.ini

```diff
@@ -63,6 +63,11 @@ clientSecret = ""
 ; end. Leave this commented out to omit OPAC links.
 ;opacURL = "http://koha-server/cgi-bin/koha/opac-detail.pl?biblionumber=%%id%%"
 
+; If we trust our authentication source and know it to be the same as the one used by
+; Koha then we can choose to not validate our patron's passwords (Useful if you are
+; using SAML/Shibboleth for authentication for both VuFind and Koha)
```

Comment:
It is probably worth adding a note here like:

```
; It is strongly recommended that you set allowUserLogin to false in the [Catalog] section
; of config.ini when setting this value to true.
```

...additionally, I assume the default value here should be false rather than true.

Reply:
I've just tested this, and it doesn't seem to have any adverse effects. Will push a commit shortly :-)

Reply:
Thanks! And just for context: if you leave allowUserLogin set to true, and the credentials provided by Shibboleth/SAML are somehow incorrect, there's a possibility that VuFind will prompt the user to enter ILS credentials... and if password validation is disabled, it would be easy to spoof other users through this form. By disabling the setting, you ensure that users can ONLY access the ILS functionality by receiving valid attributes via SSO.